moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -162,6 +162,10 @@ schemas["testpilottest"]                        = schemas.vacuous
 schemas["malware-addon-states"]                 = schemas.vacuous
 schemas["sync"]                                 = schemas.vacuous
 schemas["outofdate-notifications-system-addon"] = schemas.vacuous
+schemas["x-shield-studies"]                     = schemas.vacuous
+schemas["shield-study"]                         = schemas.vacuous
+schemas["x-contextual-feature-recommendation"]  = schemas.vacuous
+schemas["FirefoxOS"]                            = schemas.vacuous
```

Comment:
To be clear, I was not a -1. I was simply of the opinion that this shouldn't be my decision.